clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/Keys.java

```diff
@@ -86,4 +92,8 @@ public static String getClusterKey(String account, String region, String cluster
   public static String getTaskKey(String account, String region, String taskName) {
     return ID + ":" + Namespace.SERVICES + ":" + account + ":" + region + ":" + taskName;
   }
+
+  public static String getContainerInstanceKey(String account, String region, String containerInstanceArn) {
+    return ID + ":" + Namespace.CONTAINER_INSTANCES + ":" + account + ":" + region + ":" + containerInstanceArn;
```

Comment:
You could extract ":" to a variable called SEPARATOR.  not a hard requirement though

Reply:
Sure, added.